### ISSUE TITLE
Email from Microsoft Yammer causes mail to go unresponsive for a while

### ISSUE THREAD

**Issue by malwcal** [apparition47/MailTrackerBlocker#67]
When I preview mail messages that have been sent from our microsoft yammer service, mail becomes unresponsive for quite a while.  I see the beach ball, and the mac says "application not responding".

I am happy to send through via email an example of such a message.

**Comment by apparition47** [comment]
Which Mac model are you running? Are you running other plugins? The issue only happens with these Yammer emails?

Feel free to forward the email to: @apparition47 (email address in profile link, left side)

**Comment by malwcal** [comment]
This is on my MacBook Pro (15-inch, 2016), 2.7 GHz Quad-Core Intel Core i7, 16 GB 2133 MHz LPDDR3

I would not say its exclusive to those Yammer emails, but it is certainly the most notable case.  The yammer emails make mail go unresponsive for approximately 30 seconds!  Other email are probably more around the 1 second mark.

I will forward you a copy of one of the yammer email messages that trigger this.

**Comment by apparition47** [comment]
Odd. Hope it's not related to MailTrackerBlocker. Maybe try disabling it then trying it again?

**Comment by apparition47** [comment]
Will have a look if you can send me that `.eml` file

**Comment by apparition47** [comment]
```objc
@"Oracle": @[
    @"[a-zA-Z0-9-.]+/e/FooterImages/FooterImage"
],
```
Your email has giant base64 strings in it which was wrecking havoc with this particularly slow regex pattern. I'll remove this pattern for now.

**Comment by apparition47** [comment]
To be released as 0.3.19 in a bit
